src/docs/user/references/Notification-service.adoc

```diff
@@ -54,6 +54,8 @@ Users can have multiple job subscriptions with different granularity on the work
 
 At least one field of <<_glossary_workflow,*Workflow*>> data must be provided to create the subscription, that means it
 is not possible to create a job subscription that will listen for all jobs submitted to the Scheduler.
+Note that a workflow name that contains a variable i.e. my_workflow_${my_var} can be replaced by its value in case this
```

Comment:
Note that a workflow name **which** contains a variable **(e.g.,** my_workflow_${my_var} **or my_workflow_$my_var)** **is** replaced by substituting the variable with its value in case this variable is defined at the workflow level.